diffusion/models/models.py

```diff
@@ -605,3 +621,29 @@ def __call__(self, prompt, padding, truncation, return_tensors, max_length=None)
         for key in tokenized_output.keys():
             tokenized_output[key] = [tokenized_output[key], tokenized_output_2[key]]
         return tokenized_output
+
+
+def make_lora_layers(unet, rank):
+    """From https://huggingface.co/docs/diffusers/training/lora."""
+    lora_attn_procs = {}
+    for name in unet.attn_processors.keys():
```

Comment:
Do the `attn_processors` include the self attention layers or are those contained in a separate attribute? Curious if this is only adjusting cross attention layers (as intended).

Reply:
I'm not sure, but this is the way huggingface suggests LoRA be done (see the link) so I think it is safe to do it this way.